src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -0,0 +1,490 @@
+                            v-model="shareStructure.currency"
+                            :rules="getCurrencyRule()"
+                            item-text="`${data.item.name}, ${data.item.code}`"
+                            item-value="code"
+                            id='class-currency'>
+                            <template slot="selection" slot-scope="data">
+                               {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                            <template slot="item" slot-scope="data">
+                              {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                          </v-select>
+                        </v-col>
+                      </v-row>
+                    </template>
+                  </v-radio>
+                  <v-radio :value="true" label="No par value" id="radio-no-par"/>
+                </v-radio-group>
+
+                <div v-show="isSeries">
+                    <v-row v-if="shareStructure.hasParValue">
+                        <v-col cols="6">
+                            <v-text-field
+                            label="Par Value"
+                            id="series-par-value"
+                            :value="shareStructure.parValue"
+                            :disabled="true"
+                            width="10"/>
+                        </v-col>
+                        <v-col cols="6">
+                            <v-text-field
+                            id="series-currency"
+                            label="Currency"
+                            :value="`${getCurrencyNameByCode(shareStructure.currency)} (${shareStructure.currency})`"
+                            :disabled="true"/>
+                        </v-col>
+                    </v-row>
+                    <v-label id='lbl-no-par' v-else>No par value</v-label>
+                </div>
+
+                <v-divider class="separator" />
+
+                <div class="form__row">{{hasSeriesShares}}
+                  <v-checkbox
+                     id="special-rights-check-box"
+                    :label="'This share ' + shareStructure.type.toLowerCase() + ' has special rights or restrictions'"
+                    v-model="shareStructure.hasRightsOrRestrictions"
+                    @click="confirmSeriesRemoval"
+                    />
+                </div>
+
+                <div class="form__row form__btns">
+                  <v-btn large outlined color="error" :disabled="activeIndex === -1"
+                    @click="removeShareStructure()" id="remove-btn">Remove</v-btn>
+
+                  <v-btn
+                    large
+                    id="done-btn"
+                    class="form-primary-btn"
+                    color="primary"
+                    @click="validateForm()"
+                    :disabled="!formValid"
+                  >
+                    Done
```

Comment:
No change required, but if you're wondering why we somethings use `<span>` around button text, it's to make it easier to fetch just the text in the unit tests... otherwise the text may contain some line returns (so you have to "expect" that the string "contains" the string instead of "equals" the string). FYI

Reply:
I was wondering about this actually!

Reply:
Or you can put the text on the same line as the button element tags, like Vysakh did this morning.

Reply:
kk sounds good. I will do this when i finish updating the unit tests ( I'm thinking separate PR at this point 😁 )

Reply:
np